packages/reference/src/common/EntityStore.tsx

```diff
@@ -603,12 +604,14 @@ const [InternalServiceProvider, useFetch, useEntityLoader, useCurrentIds] = cons
       }
 
       const subscribeQuery = ({ queryKey, queryHash }: Query) => {
-        const [entityType, entityId] = queryKey;
```

Comment:
Every time we touch the entity stor,e something breaks. Let's hope this won't cause any issues 😅